src/server/config/mongoose.js

```diff
@@ -5,6 +5,6 @@ import '../models/users.model';
 
 export default function () {
   mongoose.Promise = global.Promise;
-  const db = mongoose.connect('mongodb://localhost/yoda');
+  const db = mongoose.connect('mongodb://localhost:27017/yoda');
```

Comment:
Why this port #? Is this the default port # for mongodb? If not, we should also update the command to bring up mongodb in the wiki or the README.md file I added to src/